src/Data/Avro/Encode.hs

```diff
@@ -72,81 +73,88 @@ newSyncBytes = BL.pack . DL.take 16 . randoms <$> initTFGen
 
 -- |Encode chunks of objects into a container, using 16 random bytes for
 -- the synchronization markers.
```

Comment:
Call out `@nullCodec@` and `@deflateCodec@` by name in the haddocks here?

Reply:
Done